src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1874,7 +1874,7 @@
 
   <AbstractNode Name="BoundCollectionExpressionBase" Base="BoundExpression">
     <!-- Collection expression elements. -->
-    <Field Name="Elements" Type="ImmutableArray&lt;BoundExpression&gt;"/>
+    <Field Name="Elements" Type="ImmutableArray&lt;BoundNode&gt;"/>
```

Comment:
Would it be worth it to introduce an `IBoundCollectionExpressionElement` interface, so we can be sure we enumerate all types that can be in this list?

Reply:
Currently, the interface would likely be implemented by `BoundCollectionExpressionSpreadElement` and `BoundExpression` so there may not be a significant benefit. Let's add an interface later as needed.